src/utility/concurrentlyArgumentsHelper.js

```diff
@@ -16,7 +16,7 @@ exports.convertTaskConfigurationToConcurrentlyArguments = function(
     );
 
     const DEFAULT_OPTIONS = {
-        killOthers: ['failure'],
+        killOthers: ['failure', 'success']
```

Comment:
Wat houdt dit in? Op het moment dat een van de taken success is, de rest killen?

Reply:
> killOthers: an array of exitting conditions that will cause a process to kill others. Can contain any of success or failure.

Hoe ik het begrijp is dat wanneer 1 proces gaat hij dan ook de taken die success hebben mag afbreken en hij dus niet doorloopt. En dat ook alles gerevert word. Maar staat niet heel duidelijk omschreven. Zag in een voorbeeld dat ze dit ook gebruikte voor een watch taak.

Reply:
Als je het dan vergelijkt in de code met de argument opties die je hebt leggen ze het als volgt uit:

--kill-others = kill other processes if one exits or dies -> dit is gelijk aan `['success, 'failure']`
--kill-others-on-fail  kill other processes if one exits with non zero status code -> gelijk aan `['failure']`

Maar is mij nog steeds niet heel veel duidelijker.

Reply:
ALs ik het zo zie dan zou het toch juist `['failure']` moeten zijn?

Reply:
ja misschien heb je wel gelijk, vage optie. Zal het terug zetten bij beide situaties was het probleem opgelost.